### DISCUSSION TITLE
Error adding Alchemy mainnet timestamp 6:01:56

### DISCUSSION THREAD

**Discussion by ArcusConsulting** [smartcontractkit/full-blockchain-solidity-course-py#417] [Q&A] [answered]
When I ran the following command:

brownie networks add development mainnet-fork -dev cmd=ganache-cli host=http://127.0.0.1 fork=https://eth-mainnet.alchemyapi.io/v2/8Pi....XXXX accounts=10 mnemonic=brownie port=8545

I get the following output:

![image](https://user-images.githubusercontent.com/91853555/142779424-e4d32c09-fd92-409b-b805-e82b71ededbd.png)

It's not an error message... but I'm not sure why I'm seeing this. Any help is appreciated, as always (looking at you, @cromewar ;) )

Thanks!

Eric

**Comment by ArcusConsulting** [comment, marked as answer]
Ah... Had an errant space in 'mainnet-fork-dev'. Solved!

**Reply by cromewar** [reply]
I just read the issue, sorry for the delay. Anyways I'm glad you solved your problem.
